### PR TITLE
Add SSDT-NVMe.dsl 

### PR DESCRIPTION
**Pull request by wy414012**
Add SSDT-NVMe.dsl to the working directory to solve the problem of external display of nvme disks.

### CODE REVIEW

**Comment by 1alessandro1** [comment]
@wy414012 it's enough to add the DeviceProperty `built-in` DATA `01` to the pciroot path of the NVMe in question without the use of any SSDT

File: Docs/AcpiSamples/Source/SSDT-NVMe.dsl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    Scope (\_SB.PCI0.NPE1)
+    {
+        Device (NVME)
+        {
+            Name (_ADR, Zero)  // _ADR: Address
+            Name (_SUN, One)  // _SUN: Slot User Number
+            Method (_INI, 0, NotSerialized)  // _INI: Initialize
+            {
+                \_SB.PCI0.NPE1.H000._ADR = 0x0F
```
Comment (vit9696) [inline Docs/AcpiSamples/Source/SSDT-NVMe.dsl:34]: This is something custom. It is not needed.

File: Docs/AcpiSamples/Source/SSDT-NVMe.dsl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                {
+                    Return (Buffer (One)
+                    {
+                         0x03                                           
+                    })
+                }
+
+                Return (Package (0x02)
+                {
+                    "built-in",
```
Comment (vit9696) [inline Docs/AcpiSamples/Source/SSDT-NVMe.dsl:49]: The value should be 00, but you are best to inject it with DeviceProperties. Only the ACPI node itself should be present in the table.

Comment (Andrey1970AppleLife) [reply Docs/AcpiSamples/Source/SSDT-NVMe.dsl:49]: > Only the ACPI node itself should be present in the table.

The device is present always in ACPI, as NVME slot or PCIE slot.

File: Docs/AcpiSamples/Source/SSDT-NVMe.dsl (hunk 1 above)
Comment (wy414012) [reply Docs/AcpiSamples/Source/SSDT-NVMe.dsl:34]: The SSDT is submitted after verification of most models. Compared with config.plist, the middle parameter resolution should be a more convenient solution.

**Pull request closed without merging**